sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -81,7 +81,7 @@ Both the asynchronous and synchronous Event Hub producer and consumer clients ca
 
 The snippet below creates a synchronous Event Hub producer.
 
-<!-- embedme ./src/samples/java/com/azure/messaging/eventhubs/ReadmeSamples.java#L31-L35 -->
+<!-- embedme https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs/ReadmeSamples.java#L31-L35 -->
```

Comment:
@sima-zhu  how did you find these links? I wouldn't have expected them to be flagged by our checks as they aren't real links from an html prospective.

Reply:
This is where we do match and replace. Will fix it.